src/Entity/Controller/SlotListBuilder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity\Controller;
+
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Config\Entity\ConfigEntityListBuilder;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Builds a listing of slot entities.
+ */
+class SlotListBuilder extends ConfigEntityListBuilder {
+
+  /**
+   * The Lift API Helper.
+   *
+   * @var \Acquia\LiftClient\Lift
+   */
+  protected $liftClient;
+
+  /**
+   * Constructs an IndexListBuilder object.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeInterface $entity_type
+   *   The entity type definition.
+   * @param \Drupal\Core\Entity\EntityStorageInterface $storage
+   *   The entity storage class.
+   * @param  \Drupal\acquia_lift\Service\Helper\LiftAPIHelper
+   *   The Lift API Helper.
+   */
+  public function __construct(EntityTypeInterface $entity_type, EntityStorageInterface $storage) {
+    parent::__construct($entity_type, $storage);
+    try {
+      /** @var \Drupal\acquia_lift\Service\Helper\LiftAPIHelper $liftHelper */
+      $liftHelper = \Drupal::getContainer()
+        ->get('acquia_lift.service.helper.lift_api_helper');
```

Comment:
Hey thank you for attempting on this issue. Yeah resolving dependency to container has always been one of my bigger challenges in the past too. I think the solution is great for now, and I wouldn't bother with requesting anything more. Please consider this thread resolved.